pkg/tcl/apitcl/v1/testworkflowmetrics.go

```diff
@@ -178,10 +178,12 @@ func (s *apiTCL) sendRunWorkflowTelemetry(ctx context.Context, workflow *testwor
 	}
 
 	var dataSource string
-	if len(workflow.Spec.Content.Files) != 0 {
-		dataSource = "files"
-	} else if workflow.Spec.Content.Git != nil {
-		dataSource = "git"
+	if workflow.Spec.Content != nil {
+		if len(workflow.Spec.Content.Files) != 0 {
+			dataSource = "files"
+		} else if workflow.Spec.Content.Git != nil {
+			dataSource = "git"
+		}
 	}
```

Comment:
This is not needed - @vLia is actually fixing that in https://github.com/kubeshop/testkube/pull/5277

Reply:
reverted